src/test/factories/responses/gitLab/listRepoCommitsResponseFactory.ts

```diff
@@ -0,0 +1,13 @@
+import { BitbucketCommit } from '../../../../services';
+import Bitbucket from 'bitbucket';
+
+export const bitbucketListCommitResponseFactory = (items: Bitbucket.Schema.Commit[]): BitbucketCommit => {
```

Comment:
Bitbucket factory in Gitlab folder? (`src/test/factories/responses/gitLab/listRepoCommitsResponseFactory.ts`)